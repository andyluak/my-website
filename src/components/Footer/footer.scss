footer {
	display:flex;
	flex-direction:column;
	justify-content: center;
	align-items: center;
	h4 {
		font-size: 1.5em;
	}

	.social_links {
		display:flex;
		gap: 2em;
		align-items: center;
		padding-bottom: 1em;
	}
}

@media screen and (max-width: 768px) {
	footer {
		h4 {
			font-size: 1em;
		}
		.social_links .social-icon {
			width: 35px;
		}
	}
}