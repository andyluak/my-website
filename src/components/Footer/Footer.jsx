import './footer.scss';

import React from 'react';
import { ReactComponent as Github } from '../../assets/images/github.svg';
import { ReactComponent as Linkedin } from '../../assets/images/linkedin.svg';
import { ReactComponent as Instagram } from '../../assets/images/instagram.svg';

const Footer = () => {
	return (
		<footer>
			<h4>Follow me on all the socials</h4>
			<div className="social_links">
				<a
					href="https://github.com/andyluak"
					target="_blank"
					rel="noopener noreferrer"
				>
					<Github className="social-icon" />
				</a>
				<a
					href="https://www.linkedin.com/in/alex-tirim-155a95192/"
					target="_blank"
					rel="noopener noreferrer"
				>
					<Linkedin className="social-icon" />
				</a>
				<a
					href="https://www.instagram.com/andylukak"
					target="_blank"
					rel="noopener noreferrer"
				>
					<Instagram className="social-icon" />
				</a>
			</div>
		</footer>
	);
};

export default Footer;
