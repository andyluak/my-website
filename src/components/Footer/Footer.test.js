import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Footer from './Footer';

describe( 'Footer component tests', () => {
	it( 'excepct to render Footer component', () => {
		expect(shallow(<Footer />)).toMatchSnapshot();
	})
})

