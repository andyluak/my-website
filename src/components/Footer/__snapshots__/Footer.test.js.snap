// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component tests excepct to render Footer component 1`] = `
<footer>
  <h4>
    Follow me on all the socials
  </h4>
  <div
    className="social_links"
  >
    <a
      href="https://github.com/andyluak"
      rel="noopener noreferrer"
      target="_blank"
    >
      <ForwardRef(SvgGithub)
        className="social-icon"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/alex-tirim-155a95192/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <ForwardRef(SvgLinkedin)
        className="social-icon"
      />
    </a>
    <a
      href="https://www.instagram.com/andylukak"
      rel="noopener noreferrer"
      target="_blank"
    >
      <ForwardRef(SvgInstagram)
        className="social-icon"
      />
    </a>
  </div>
</footer>
`;
